<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>640</width>
    <height>480</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Segoe UI</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>Sms On PC</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QFrame#frame_1, QFrame#frame_parametres{
	border-right: 1px solid black;
}

QFrame#frame_2{
	background-image: url(:/arriere/Graphics/Wallpapers/forest_background.png);
}

QLineEdit#lEdit_recherche{
	border: 2px solid  #FFCCBC;
	border-radius: 10px;
}

QToolButton#tBtn_parametres, QToolButton#tBtn_emoji{
	border: 0px;
	border-radius: 50px;
}

QTextEdit#tEdit_message{
	margin: 0px 0px 5px 5px;
	border: 2px solid #FFCCBC;
	background-color: white;
}

QListWidget::item{	
	color: #212121;
	padding-top: 2px;
	padding-bottom: 2px;
	font-size: 24px;
	border: 0px;
}

QListWidget::item:selected, QListWidget::item:focus, QListWidget::item:focus QLabel{	
	background-color:#FF5722;
	color: white;
	border: 0px;
}

QScrollBar{
	width: 10px;
}</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QFrame" name="frame_central">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>640</width>
      <height>480</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QFrame" name="frame_1">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>220</width>
       <height>480</height>
      </rect>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QFrame" name="frame_parametres">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>220</width>
        <height>80</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QToolButton" name="tBtn_parametres">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>50</width>
         <height>50</height>
        </rect>
       </property>
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="ressources.qrc">
         <normaloff>:/icones/Graphics/Icons/Settings-icon.png</normaloff>:/icones/Graphics/Icons/Settings-icon.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>50</width>
         <height>50</height>
        </size>
       </property>
      </widget>
      <widget class="QLineEdit" name="lEdit_recherche">
       <property name="geometry">
        <rect>
         <x>70</x>
         <y>20</y>
         <width>130</width>
         <height>30</height>
        </rect>
       </property>
      </widget>
     </widget>
     <widget class="QFrame" name="frame_liste">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>80</y>
        <width>220</width>
        <height>400</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QListWidget" name="listWidget">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>220</width>
         <height>400</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <pointsize>12</pointsize>
         <italic>true</italic>
        </font>
       </property>
       <property name="focusPolicy">
        <enum>Qt::NoFocus</enum>
       </property>
       <property name="frameShape">
        <enum>QFrame::NoFrame</enum>
       </property>
       <property name="lineWidth">
        <number>0</number>
       </property>
       <property name="iconSize">
        <size>
         <width>40</width>
         <height>40</height>
        </size>
       </property>
       <property name="movement">
        <enum>QListView::Free</enum>
       </property>
       <property name="layoutMode">
        <enum>QListView::SinglePass</enum>
       </property>
       <property name="viewMode">
        <enum>QListView::ListMode</enum>
       </property>
       <property name="sortingEnabled">
        <bool>false</bool>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QFrame" name="frame_2">
     <property name="geometry">
      <rect>
       <x>220</x>
       <y>0</y>
       <width>440</width>
       <height>480</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true"/>
     </property>
     <property name="frameShape">
      <enum>QFrame::NoFrame</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <widget class="QFrame" name="frame_discussion">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>40</y>
        <width>440</width>
        <height>350</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::NoFrame</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QWidget" name="verticalLayoutWidget">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>411</width>
         <height>341</height>
        </rect>
       </property>
       <layout class="QVBoxLayout" name="discussLayout"/>
      </widget>
     </widget>
     <widget class="QFrame" name="frame_message">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>400</y>
        <width>340</width>
        <height>100</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QTextEdit" name="tEdit_message">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>340</width>
         <height>80</height>
        </rect>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Expanding">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="frameShape">
        <enum>QFrame::NoFrame</enum>
       </property>
      </widget>
     </widget>
     <widget class="QFrame" name="frame_emoji">
      <property name="geometry">
       <rect>
        <x>340</x>
        <y>400</y>
        <width>80</width>
        <height>80</height>
       </rect>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QToolButton" name="tBtn_emoji">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>0</y>
         <width>80</width>
         <height>80</height>
        </rect>
       </property>
       <property name="text">
        <string>...</string>
       </property>
       <property name="icon">
        <iconset resource="ressources.qrc">
         <normaloff>:/icones/Graphics/Icons/Emoji-Blank-icon.png</normaloff>:/icones/Graphics/Icons/Emoji-Blank-icon.png</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>80</width>
         <height>80</height>
        </size>
       </property>
       <property name="shortcut">
        <string>Ctrl+R</string>
       </property>
      </widget>
     </widget>
     <widget class="QPushButton" name="psbGetContact">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Get Contact</string>
      </property>
     </widget>
     <widget class="QPushButton" name="psbAddMsg">
      <property name="geometry">
       <rect>
        <x>90</x>
        <y>10</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>Add Message</string>
      </property>
     </widget>
     <widget class="QPushButton" name="textButton">
      <property name="geometry">
       <rect>
        <x>180</x>
        <y>10</y>
        <width>75</width>
        <height>23</height>
       </rect>
      </property>
      <property name="text">
       <string>PushButton</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lneNumber">
      <property name="geometry">
       <rect>
        <x>260</x>
        <y>10</y>
        <width>141</width>
        <height>20</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="ressources.qrc"/>
 </resources>
 <connections/>
</ui>
